apps/api/src/migration/1657325456713-MapRoleAcl.ts

```diff
@@ -0,0 +1,210 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class MapRoleAcl1657325456713 implements MigrationInterface {
```

Comment:
If possible, it would be best to use the query builder to create these migrations instead just using .query.

Reply:
In my opinion, based on typeorm document. Using query is a more flexible way to play around.
Query builder is a way to use migration API. And that won't work in all the cases with generate statements(like if-exist partial unique constrain or migrate data from one form to another form).
but query works well in all the cases, by simply running `typeorm migration: generate -n <name>`. Moreover, sometimes it requires manual crossover like changing a column from array/JSON to many-to-many relation without data loss, this scenario handles pretty nicely with query.

Reply:
In some cases, query builder gets more complex than raw queries or it takes too much time. I need to study more about query builder. Anyway, I wanted to keep it consistent, so I chose to stick to raw query than mixing with query builder.
I tried to generate  a migration and it returned a bunch of queries not related to my work. Next time, I'll try and pick just what I need from the generated.